### ISSUE TITLE
Import Project from SynBioHub

### ISSUE THREAD

**Issue by zachzundel** [MyersResearchGroup/iBioSim#464]
I think this could be done if you added an 'import from SynBioHub' function that would create a new project directory. You'd just have to also display a filesystem prompt to ask where the project should be placed. 

Alternatively, I think it might be useful to rethink the way that projects work in iBioSim, and have a user pick a workspace (à la Eclipse) and then create a folder in that workspace for each new project -- that would mean that no file prompt would be needed to import a new project.

**Comment by cjmyers** [comment]
Agreed.  There is likely an issue buried deep to this effect.  I’ve long thought it would be best to have a single workspace for projects.
